package.json

```diff
@@ -33,13 +38,29 @@
     "request": "^2.88.2",
     "slug": "^8.2.2",
     "socket.io": "^4.6.1",
+    "uglifycss": "^0.0.29",
     "web-push": "^3.6.1",
-    "xss": "^1.0.14"
+    "xss": "^1.0.14",
+    "zlib": "^1.0.5"
   },
   "devDependencies": {
```

Comment:
er zijn paar packages die we niet gebruiken

Reply:
Klopt ook!